shoop_tests/admin/test_user_module.py

```diff
@@ -41,6 +41,13 @@ def test_user_detail_works_at_all(rf, admin_user):
 def test_user_detail_contact_seed(rf):
     get_default_shop()
     contact = create_random_person()
+
+    # Using random data for name and email would need escaping when
+    # checking if it is rendered, therefore use something very basic instead
```

Comment:
Because there is no point in having random data like this in the tests. What are we trying to test here anyway?